docs/src/public/userguide/pages/tds_tutorial/tds_configuration/CustomizingTds.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Customizing TDS
+last_updated: 2018-06-30
+sidebar: tdsTutorial_sidebar
+toc: false
+permalink: /customizing_tds.html
+---
+
+TDS provides a customizable user interface using [Thymeleaf](https://www.thymeleaf.org/) Java template engine.
+The pages which currently support customization are:
+ * Catalog
+ * Dataset access
+
+UI customization can be implemented through the contribution of both CSS stylesheets and Thymeleaf HTML templates.
+
+## CSS Stylesheets
+
+Custom stylesheets are specified in the `threddsConfig.xml` configuration file.
+There are three properties within the `htmlSetup` element used to define stylesheets:
+
+~~~xml
+<htmlSetup>
+   <standardCssUrl>standard.css</standardCssUrl>
+   <catalogCssUrl>catalog.css</catalogCssUrl>
+   <datasetCssUrl>dataset.css</datasetCssUrl>
+</htmlSetup>
+~~~
+
+Values given for `catalogCssUrl` and `datasetCssUrl` will be applied as stylesheets on the Catalog and Dataset
+access HTML pages, respectively. A path supplied to `standardCssUrl` will be used in all generated HTML pages.
+
+Note: Paths to CSS files may be absolute or relative, relative paths must be relative to the webapp URL,
+ i.e. http://server:port/thredds/.

```

Comment:
It might be good to add that if the css files defined in `threddsConfig.xml` are located in `tds.content.root.path/thredds/public` directory, everything should work. So, by putting a file called `standard.css` inside `tds.content.root.path/thredds/public`, no changes to threddsConfig.xml would need to be made and the css file would be used.

That said, how would someone add a css file such that it would show up relative to `http://server:port/thredds/`? Would they need to copy it into the WEB-INF directory of the exploded war file?

Reply:
I copied that out of the comments in threddsConfig, I think it was outdated instructions; I've deleted it from both places and replaced it with instructions to use the public directory.

As of now, they do have to put the names of the CSS files they want to use in threddsConfig.xml, but we could instead search the public directory for expected filenames - might make more sense since the only benefit the threddsConfig property adds is the ability to name the files whatever they want